vk-video/build.rs

```diff
@@ -0,0 +1,13 @@
+fn main() {
+    cfg_aliases::cfg_aliases! {
+            vulkan: {
```

Comment:
fix formating

Reply:
unfortunately, that's how `cargo fmt` formats it.

Reply:
Are you sure? I tried to change it locally and fmt does not affect it. Plus it's a macro from my experience formatter does not really touch it?

Reply:
I fixed it manually and `cargo fmt` in our CI detected it as unformatted: https://github.com/software-mansion/live-compositor/actions/runs/11218011522/job/31180837362

Reply:
maybe you run `cargo fmt` without the `--all` flag?

Reply:
no, but I have older rust than in CI, so maybe that is the reason